blog/2022/06-22-kubernetes-support/index.md

```diff
@@ -0,0 +1,275 @@
+kubernetes_validating_webhook_configuration: 4
+kubernetes_priority_class: 4
+kubernetes_pod_disruption_budget: 5
+kubernetes_volume_attachment: 6
+kubernetes_node: 6
+kubernetes_csi_node: 6
+kubernetes_persistent_volume_claim: 7
+kubernetes_persistent_volume: 7
+kubernetes_stateful_set: 7
+kubernetes_daemon_set: 11
+kubernetes_priority_level_configuration: 14
+kubernetes_namespace: 14
+kubernetes_flow_schema: 22
+kubernetes_deployment: 25
+kubernetes_role: 31
+kubernetes_role_binding: 32
+kubernetes_controller_revision: 37
+kubernetes_endpoint_slice: 45
+kubernetes_service: 45
+kubernetes_endpoint: 47
+kubernetes_replica_set: 63
+kubernetes_pod: 70
+kubernetes_config_map: 73
+kubernetes_service_account: 122
+kubernetes_secret: 160
+kubernetes_cluster_role_binding: 166
+kubernetes_cluster_role: 193
+total matched: 1234
+total unmatched: 0
+```
+
+How many pods are running on each node in the cluster?
+
+```bash
+> search is(kubernetes_pod) | count /ancestors.kubernetes_node.reported.name
+
+helm2-cjhg9: 7
+analytics-u1wtx: 11
+helm2-cjhgz: 11
+analytics-u1wtj: 11
+analytics-u1wto: 15
+helm2-cjhg1: 15
+total matched: 70
+total unmatched: 0
+```
+
+Reveal general information about the Kubernetes clusters, including version and server URL:
+
+```bash
+> search is(kubernetes_cluster) | list cluster_info
+
+major=1, minor=21, platform=linux/amd64, server_url=https://31e9afd3-xxxxx.k8s.ondigitalocean.com
+major=1, minor=21, platform=linux/amd64, server_url=https://9a3ac2b5-xxxxx.k8s.ondigitalocean.com
+```
+
+Show detailed information about a Kubernetes service with name `resotocore`:
+
+```yaml
+> search is(kubernetes_service) and name~resotocore | dump
+
+id: AS-E-YjvAftQUcuUmNC52g
+reported:
+  ctime: '2022-04-20T14:52:58Z'
+  id: df8f6c37-5542-4e17-b2f4-bb6caaa2ba17
+  labels:
+    app.kubernetes.io/instance: resoto
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: resoto
+    app.kubernetes.io/version: 2.2.0
+    helm.sh/chart: resoto-0.2.0
+    resoto: core
+  name: resoto-resotocore
+  namespace: resoto
+  resource_version: '51578078'
+  service_spec:
+    cluster_ip: 10.245.133.206
+    cluster_ips:
+    - 10.245.133.206
+    external_ips: []
+    external_traffic_policy: Cluster
+    ip_families:
+    - IPv4
+    ip_family_policy: SingleStack
+    load_balancer_source_ranges: []
+    ports:
+    - name: http
+      node_port: 32648
+      port: 8900
+      protocol: TCP
+      target_port: api
+    session_affinity: None
+    type: LoadBalancer
+  service_status:
+    conditions: []
+    load_balancer:
+      ingress:
+      - ip: 132.159.204.142
+        ports: []
+  tags:
+    kubernetes.digitalocean.com/load-balancer-id: 03d5ed66-fefe-41fb-a3f6-9a0e43560adb
+    meta.helm.sh/release-name: resoto
+    meta.helm.sh/release-namespace: resoto
+  kind: kubernetes_service
+  age: 1mo29d
+```
+
+Use fulltext search to find resources, that match any property no matter where it is defined:
```

Comment:
I think Fulltext search deserves a call out as a specific feature and explain how it works, before you show how to use it. it's one of our biggest differentiators. 

I'd also include a link to the docs: https://resoto.com/docs/concepts/search